iphone/Classes/GeolocationModule.m

```diff
@@ -848,6 +854,35 @@ - (void)requestLocationPermissions:(CLAuthorizationStatus)authorizationType with
   }
 }
 
+#if IS_SDK_IOS_14
+- (void)requestTemporaryFullAccuracyAuthorization:(NSString *)purposeKey withCallback:(JSValue *)callback
+{
+  if (![TiUtils isIOSVersionOrGreater:@"14.0"]) {
+    DebugLog(@"[WARN] This is available in iOS 14.0+ only");
```

Comment:
fire the callback with an error-style event? i.e. code of 1, message stating "Supported on iOS 14+"?